ncdump/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@
 # 2015, 2016, 2017, 2018
 # University Corporation for Atmospheric Research/Unidata.
 
-# See netcdf-c/COPYRIGHT file for more info.
+# See netcdffff-c/COPYRIGHT file for more info.
```

Comment:
I'll fix this typo, so far it's the only thing I've found that obviously needs to be fixed :)